metrics-core/src/main/java/com/codahale/metrics/CsvReporter.java

```diff
@@ -124,6 +125,7 @@ public CsvReporter build(File directory) {
     private final File directory;
     private final Locale locale;
     private final Clock clock;
+    private final HashMap<File, PrintWriter> printWriterHashMap = new HashMap<File, PrintWriter>();
```

Comment:
`CsvReporter` extends a `ScheduledReporter` class, which as we can see on [this line](https://github.com/dropwizard/metrics/blob/3.2-development/metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java#L79), only uses a `SingleThreadScheduledExecutor`, so to me it doesn't seem like an issue. If you still think this could be an issue, please let me know! 😄